methods.go

```diff
@@ -352,7 +352,7 @@ func (c *Client) RemoveAccount(username string) (bool, error) {
 	var args rencode.List
 	args.Add(username)
 
-	// perform login
+	// remove account
```

Comment:
This is correct now, but it became a little redundant. Should we remove it?

Reply:
Yes